source/gems/dry-container/registry-and-resolver.html.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Registry &amp; Resolver
+layout: gem-single
+---
+
+### Register options
+
+#### `call`
+
+This option register container wrapped (or not) to proc:
```

Comment:
Think this would be clearer as `This boolean option determines whether or not the registered item should be invoked when resolved, i.e.` or something along those lines